src/components/Typography/Link/Link.tsx

```diff
@@ -0,0 +1,86 @@
+import { ReactEventHandler } from "react";
+import { Icon } from "@/components/Icon/Icon";
+import styled from "styled-components";
+
+type TextSize = "xs" | "sm" | "md" | "lg";
+type TextWeight = "normal" | "medium" | "semibold" | "bold";
+
+export type LinkProps = {
+  size?: TextSize;
+  weight?: TextWeight;
+  className?: string;
+  href?: string;
+  onClick?: ReactEventHandler;
+  isExternal?: boolean;
+  children?: React.ReactNode;
+};
+
+/** Component for linking to other pages or sections from with body text */
+export const Link = ({
+  size = "md",
+  weight = "normal",
+  className,
+  href,
+  onClick,
+  isExternal = false,
+  children,
+}: LinkProps) => (
+  <CuiLink
+    size={size}
+    weight={weight}
+    className={className}
+    href={href}
+    onClick={onClick}
+    target={isExternal ? "_blank" : undefined}
+  >
+    {children}
+    {isExternal && (
```

Comment:
Should we use isExternal sometimes we want the external link and not use the icon

Reply:
Yep, see https://github.com/ClickHouse/click-ui/pull/92#discussion_r1294292913